pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -28,9 +28,9 @@ include::modules/ref_prerequisites.adoc[leveloffset=+1]
 [id="{ProductShortName}-concepts"]
 == {ProductShortName} Concepts
 
-// include::modules/<pending>
+include::modules/con_module-metadata-in-pantheon.adoc[leveloffset=+2]
```

Comment:
Looks like this file `con_module-metadata-in-pantheon.adoc` also needs to be added to the PR, it is not present on the master branch in /modules